src/components/ProfilePage.js

```diff
@@ -32,53 +67,85 @@ export default function ProfilePage() {
         
     }, []);
 
+    const editProfile = async (e) => {
+        e.preventDefault();
+        setError(false); 
+        setErrorMessage(''); 
+            axiosInstance
+                .put(`/users/${user}` , {
+                    headers: {authorization} , 
+                    firstName: firstName,
+                    lastName : lastName,
+                    email : email,
+                    gender : gender,
+                    bio: bio,
+                    occupation: occupation,
+                    links: links,
+                    interests: interests,
+                    skills: skills
+                
+                })
+                .then((response) => {
 
-    return (
-        
+                    alert("Profile was edited successfully!");
+                    navigate(`/users/${user}`);
+                    console.log(response);
+                    
+            
+                })
+                .catch((err) => {
+                    console.log(err);
+                    console.log(err.response);
+                    setErrorMessage(err.response.data.msg); 
+                    setError(true);
+                });
+         
+    };
 
-        <Container>
+   
+
+    return (
 
+        <section id="header">
             
-            <div>
-            EDIT PROFILE
-        
-            </div>         
+        <Container>
+                   
           <Row>
                 <Col>
                 
                 <Form>
                 <Form.Group>
                 <Form.Label>First Name</Form.Label>
-                <Form.Control type="text"  readOnly />
+                <Form.Control type="text" defaultValue = {firstName} readOnly />
                 <Form.Label>Last Name</Form.Label>
-                <Form.Control type="text" placeholder="Readonly input here..." readOnly />
+                <Form.Control type="text" defaultValue = {lastName} readOnly />
                 <Form.Label>Gender</Form.Label>
-                <Form.Control type="text" placeholder="Readonly input here..." readOnly />
+                <Form.Control type="text" defaultValue = "Male" readOnly />
                 </Form.Group>
                 <Form.Group className="mb-3" controlId="exampleForm.ControlInput1">
                 <Form.Label>Email</Form.Label>
-                <Form.Control type="text" placeholder="Readonly input here..." readOnly />
+                <Form.Control type="email" defaultValue = {email} readOnly />
                 </Form.Group>
                 <Form.Group className="mb-3" controlId="exampleForm.ControlInput1">
                 <Form.Label>Occupation</Form.Label>
-                <Form.Control type="email" placeholder="name@example.com" />
+                <Form.Control type="text" defaultValue = {occupation} onChange = {(e) => (setOccupation(e.target.value))}/>
```

Comment:
How come there's an `onChange` handler here but not for first name, last name, and email?